components/http/okHttp/src/main/java/com/microsoft/kiota/http/KiotaClientFactory.java

```diff
@@ -84,6 +87,7 @@ private KiotaClientFactory() {}
      */
     @Nonnull public static Interceptor[] createDefaultInterceptors() {
         return new Interceptor[] {
+            new UrlReplaceHandler(),
```

Comment:
This doesn't perform any action when configured by default. I think we could leave it out and allow developers to add it when they need to configure it to perform a replacement

Reply:
this method should call into the one that accepts options and pass the options to avoid duplicated definitions.